javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,9 @@
+const firstWords = ["Easy", "Awesome", "Corporate", "Bright", "Advanced"];
+const secondWords = ["Life", "Action", "Together", "Way", "Future"];
+
+const startupName = ;
+
+function startupName () {
+  Math.floor(Math.random())
+  console.log(startupName(firstWords, secondWords))
+};
```

Comment:
Good try at using functions in the week 1 homework before we learned them! 

You can't use a function inside itself, so this wouldn't work. 

If I were to re-write it, it might look something like: 
```js
function getStartUpName(firstWords, secondWords) {
    const firstName = firstWords[Math.floor(Math.random() * firstWords.length)]
    const secondName = secondWords[Math.floor(Math.random() * firstWords.length)]
    return firstName + ' ' + secondName
}

const startupName = getStartUpName(["Easy", "Awesome", "Corporate", "Bright", "Advanced"], ["Life", "Action", "Together", "Way", "Future"])

console.log(startupName)
```